### PR TITLE
Update Digital Ocean instruction for Sync Endpoint with Cloud Services

### PR DESCRIPTION
**Pull request by EstherPhebe**
closes odk-x/tool-suite-X#292

#### What is included in this PR?
1. Updated instruction for setting up a droplet (server) on the Digital Ocean platform

#### What new issues will need to be opened because of this PR?
None
#### What is left to be done in the addressed issue?
None
#### What problems did you encounter?
None

### CODE REVIEW

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 1]
```diff
@@ -55,49 +54,47 @@ Setting up a DigitalOcean account
...
+    Sync Endpoint requires more than *2GB* of space to run, this means that plans below *4GB* will not work.
+  
+  .. image:: /img/setup-digital-ocean/do-distribution.png
+    :width: 600
 
+  .. image:: /img/setup-digital-ocean/do-plan.png
+    :width: 600
 
-3. Generate API token by logging into DigitalOcean console and clicking on :guilabel:`API` under the **MANAGE** section. Now, click on :guilabel:`Generate New Token` and enter a name.
+3. Scroll down to the :guilabel:`Select additional options`, click on the User data checkbox, copy and paste the contents of the :download:`cloud_init_DO.yml</files/cloud_init_DO.yml>` file in the text area provided.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:73]: is this something you need to download? If so, suggest a "download, then copy and paste the contents."

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 2]
```diff
@@ -55,49 +54,47 @@ Setting up a DigitalOcean account
...
-   :width: 600
+  .. image:: /img/setup-digital-ocean/do-userdata.png
+    :width: 600
 
-4. Download the following :download:`pyscript_DO.py</files/pyscript_DO.py>` and :download:`cloud_init_DO.yml</files/cloud_init_DO.yml>` files we have provided and ensure that they are located in the same directory. Switch to that directory and run the following command in order to set up your droplet:
+  .. image:: /img/setup-digital-ocean/do-userdata2.png
+    :width: 600
 
-  .. code-block:: console
+4. The next step is :guilabel:`Authentication`. The are two authentication types to select from; **SSH Keys** and **Password**. We highly recommend that use an SSH key for authentication. Copy and paste your SSH key username, and the key itself.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:81]: "There are"

Comment (elmps2018) [reply odkx-src/sync-endpoint-cloud-setup.rst:81]: "recommend that you use a SSH key"

**Review by elmps2018 — COMMENTED** [review]
This looks great with the new explanations and well-marked pictures! Just a few minor edits.

File: odkx-src/sync-endpoint-cloud-setup.rst (hunk 1 above)
Comment (EstherPhebe) [reply odkx-src/sync-endpoint-cloud-setup.rst:73]: Once you click on the file, it opens in the browser. It doesn't need to be downloaded.

**Comment by elmps2018** [comment]
Note a test failed on circleci

**Comment by elmps2018** [comment]
This looks good to me now aside from circleci test fail. Definitely want @wbrunette to check as well.

**Review by wbrunette — APPROVED** [review]

**Pull request merged**